core-concepts/plugins.md

```diff
@@ -241,22 +242,22 @@ defineStore(
     // ...
   },
   {
-    // this will be read by a plugin later on
+  // Esto será leído por un plugin más adelante
     debounce: {
-      // debounce the action searchContacts by 300ms
+    // retrasa la acción searchContact durante 300ms
       searchContacts: 300,
     },
   }
 )
 ```
 
-## TypeScript
+## TypeScript {#typescript}
```

Comment:
I think it is not necessary to add an identifier to title anchors if they have not been translated.

Reply:
The reason why I believe we should put it anyway is to write all headers in the same way: # name {anchor}